frontend/src/components/NavBar/index.tsx

```diff
@@ -13,7 +13,7 @@ export const links = [
     path: '/launch',
   },
   {
-    name: 'Manager',
+    name: 'Manage',
```

Comment:
I prefer to keep an "action incentiver" wording with `Deploy` `Manage` `Launch` etc.